Projects/Shapes/Shapes/Shapes/Main.cpp

```diff
@@ -0,0 +1,8 @@
+#include "ShapeBuilder.h"
+
+
+int main()
+{
+	FShapeBuilder ShapeBuilderApp = FShapeBuilder();
```

Comment:
Remember that the `= FShapeBuilder()` is not necessary. You're calling a copy constructor and passing in a default constructed shape.

The compiler will most likely optimize it, but still, it's worth knowing about it.